fe/src/project/Seller.Portal/areas/Orders/components/EditOrderForm/EditOrderForm.js

```diff
@@ -62,44 +62,63 @@ function EditOrderForm(props) {
         <section className="section section-small-padding edit-order">
             <h1 className="subtitle is-4">{props.title}</h1>
             <h2 className="subtitle is-5 edit-order__items-subtitle">{props.orderStatusLabel}</h2>
-            <form className="is-modern-form" onSubmit={handleOrderStatusSubmit} method="post">
-                <div className="columns is-desktop">
-                    <div className="column is-3">
+            {props.id &&
+                <div className="columns is-desktop mt-0 mb-0">
+                    <div className="column is-3 pb-0">
                         <div className="field">
-                            <FormControl fullWidth={true}>
-                                <InputLabel id="order-status-label">{props.orderStatusLabel}</InputLabel>
-                                <Select
-                                    labelId="order-status-label"
-                                    id="orderStatus"
-                                    name="orderStatus"
-                                    value={orderStatusId}
-                                    onChange={(e) => {
-
-                                        e.preventDefault();
-                                        setOrderStatusId(e.target.value);
-                                    }}>
-                                    {props.orderStatuses.map(status => {
-                                        return (
-                                            <MenuItem key={status.id} value={status.id}>{status.name}</MenuItem>
-                                        );
-                                    })}
-                                </Select>
-                            </FormControl>
+                            <TextField 
```

Comment:
**suggestion:** I'd rather see it simply as a label. The TextField makes no sense since it's readonly.